### PR TITLE
Fix misspellings and minor grammar in comments

### PR DESCRIPTION
**Pull request by ghost**
Fixes misspellings and minor grammar issues in the comments. Some changes will appear in the doxygen help, I believe.

### CODE REVIEW

File: include/wx/generic/private/markuptext.h [hunk 1]
```diff
@@ -120,7 +120,7 @@ class WXDLLIMPEXP_CORE wxItemMarkupText : public wxMarkupTextBase
     // instead of generic wxDC::DrawLabel(), so is more suitable for use in
     // controls that already use DrawItemText() for its items.
     //
-    // The meaning of the flags here is different than in wxMarkupText too:
+    // The meaning of the flags here is different from in wxMarkupText too:
```
Comment (vadz) [inline include/wx/generic/private/markuptext.h]: We can't have "from in" and I don't see anything wrong with the original "than"?

File: src/msw/toplevel.cpp [hunk 2]
```diff
@@ -192,7 +192,7 @@ WXDWORD wxTopLevelWindowMSW::MSWGetStyle(long style, WXDWORD *exflags) const
             //    the taskbar normally)
             //
             // The second one is solved here by using WS_EX_APPWINDOW flag, the
-            // first one is dealt with in our MSWGetParent() method
+            // first one is dealt within our MSWGetParent() method
```
Comment (vadz) [inline src/msw/toplevel.cpp]: This one is definitely wrong, the meaning is that it is "dealt with" "in ... method". Maybe the current comment awkward but the new one is just wrong.

File: src/common/sizer.cpp [hunk 3]
```diff
@@ -2378,7 +2378,7 @@ void wxBoxSizer::RecalcSizes()
 
             wxCoord maxMajor = GetSizeInMajorDir(item->GetMaxSizeWithBorder());
 
-            // must be nonzero, fixed-size items were dealt with in previous loop
+            // must be nonzero, fixed-size items were dealt within previous loop
```
Comment (vadz) [inline src/common/sizer.cpp]: Not sure about this one... do people really say "within a loop"? I don't think so...

**Review by vadz — CHANGES_REQUESTED** [review]
Thanks for the fixes, but a few of them don't look right to me, could you please recheck? TIA!

File: src/common/addremovectrl.cpp [hunk 4]
```diff
@@ -54,7 +54,7 @@ wxAddRemoveCtrl::Create(wxWindow* parent,
         return false;
 
     // We don't do anything here, the buttons are created when we're given the
-    // adaptor to use them with in SetAdaptor().
+    // adaptor to use them within SetAdaptor().
```
Comment (catalinr) [inline src/common/addremovectrl.cpp]: This looks wrong too. The old for seems more logical: `buttons` [...] `we're given the adaptor to use them with` [where?] `in SetAdaptor()`. IOW "the buttons are created in SetAdaptor() when we're given the adaptor to use them with"

Comment (Blake-Eryx) [reply src/common/addremovectrl.cpp]: Maybe the word "in" should be stricken?  One of the words should be stricken at least.

File: src/common/sizer.cpp (hunk 3 above)
Comment (Blake-Eryx) [reply src/common/sizer.cpp]: Yes, that is correct.

File: src/msw/toplevel.cpp (hunk 2 above)
Comment (Blake-Eryx) [reply src/msw/toplevel.cpp]: I see.  Yes, "with in" is correct.

File: include/wx/generic/private/markuptext.h (hunk 1 above)
Comment (Blake-Eryx) [reply include/wx/generic/private/markuptext.h]: Yeah, guess this one's OK.

File: src/common/sizer.cpp (hunk 3 above)
Comment (Blake-Eryx) [reply src/common/sizer.cpp]: Wait, maybe "with in" is correct.  Sorry, just awkward wording I guess, will change it back...

**Comment by Blake-Eryx** [comment]
OK, I made the suggested changes, hoping these look alright now.  

I'm new to git, but I think I did this correctly.

Thanks!

File: interface/wx/timectrl.h [hunk 5]
```diff
@@ -150,11 +150,11 @@ class wxTimePickerCtrl : public wxControl
         The date part of @a dt is ignored, only the time part is displayed in
         the control. The @a dt object must however be valid.
 
-        In particular notice that it is a bad idea to use default wxDateTime
+        In particular, notice that it is a bad idea to use default wxDateTime
         constructor from hour, minute and second values as it uses the today
-        date for the date part which means that some times can be invalid if
+        date for the date part, which means that sometimes can be invalid if
```
Comment (vadz) [inline interface/wx/timectrl.h]: Sorry, but this is still wrong, "some times" here means "some wxDateTime objects".

Please do not change things when in doubt, it's better to leave a possible typo than "fix" it and change the meaning of the sentence.

**Comment by vadz** [comment]
Thanks, I'm merging this but, again, typo fixes and style improvements are definitely welcome, but not if they change the meaning of the text.

Thanks again!

**Pull request closed without merging**